app/src/main/java/org/wikipedia/diff/ArticleEditDetailsFragment.kt

```diff
@@ -448,10 +575,87 @@ class ArticleEditDetailsFragment : Fragment(), WatchlistExpiryDialog.Callback, L
             .show()
     }
 
+    private fun showRollbackSnackbar() {
+        val snackbar = FeedbackUtil.makeSnackbar(requireActivity(), getString(R.string.revision_rollback_success))
```

Comment:
Rollback feedback string seems to be different than designs. Expected: "Rollback complete" Actual: Edits reverted: https://www.figma.com/file/cJmB34iJE5p8qbqrQWnE50/Patrolling-on-Apps?type=design&node-id=1504-69602&mode=design&t=u7XLyt3NC01sAoBV-0

Reply:
Done and please see #4243